docker-compose-test-tools.yml

```diff
@@ -33,14 +33,17 @@ services:
     container_name: edgex-app-service-configurable
     hostname: app-service-configurable
     environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
       edgex_registry: consul://edgex-core-consul:8500
-      edgex_service: http://app-service-configurable:48095
       edgex_profile: blackbox-tests
       Service_Host: app-service-configurable
       Clients_CoreData_Host: edgex-core-data
       Clients_Logging_Host: edgex-support-logging
-      Logging_EnableRemote: "true"
+      Logging_EnableRemote: "false"
+      Service_Host: edgex-app-service-configurable
+      Service_Port: 48095
       MessageBus_SubscribeHost_Host: edgex-core-data
+      Binding_PublishTopic: events
```

Comment:
Not needed as the `blackbox-tests` profile used here doesn't use the Message Bus.